IdentityCore/tests/util/MSIDApplicationTestUtil.m

```diff
@@ -70,7 +86,7 @@ - (BOOL)openURL:(NSURL *)url
     return s_onOpenUrl(url, nil);
 }
 
-- (BOOL)canOpenURL:(NSURL *)url
+- (BOOL)TO_canOpenURL:(NSURL *)url
```

Comment:
Rename `TO_canOpenURL` to "msidCanOpenURL". According to our codestyle, we always start with small letter and don't use underscore.

Reply:
Will update